e2e_test/source/basic/inlcude_key_as.slt

```diff
@@ -48,9 +48,25 @@ WITH (
 	topic = 'upsert_json')
 FORMAT PLAIN ENCODE JSON
 
+statement ok
+create table additional_columns (a int)
+include key as key_col
+include partition as partition_col
+include offset as offset_col
+include timestamp as timestamp_col
+include header as header_col
```

Comment:
What's the default column name?

Reply:
Ok I find it.
https://github.com/risingwavelabs/risingwave/blob/512e88497b423a05f2eb65821f9e0c800f07ace6/src/frontend/src/handler/create_source.rs#L517-L519